phpBB/phpbb/install/console/command/converter/convert.php

```diff
@@ -0,0 +1,210 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license       GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\install\console\command\converter;
+
+use phpbb\install\exception\installer_exception;
+use phpbb\install\converter\module\converter_obtain_data\module;
+use phpbb\install\converter\controller\helper;
+use phpbb\install\helper\config;
+use phpbb\install\helper\install_helper;
+use phpbb\install\helper\iohandler\cli_iohandler;
+use phpbb\install\helper\iohandler\factory;
+use phpbb\install\helper\container_factory;
+use phpbb\install\installer_configuration;
+use phpbb\language\language;
+use Symfony\Component\Config\Definition\Exception\Exception;
+use Symfony\Component\Config\Definition\Processor;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Yaml\Exception\ParseException;
+use Symfony\Component\Yaml\Yaml;
+
+class convert extends \phpbb\console\command\command
+{
+	/**
+	 * @var factory
+	 */
+	protected $iohandler_factory;
+
+	/**
+	 * @var install_helper
+	 */
+	protected $helper;
+
+	/**
+	 * @var language
+	 */
+	protected $language;
+
+	protected $obtain_data_module;
```

Comment:
Havent generated docblocks on the command.php file since its dependencies may change. I'll add them as soon as everything is fixed.